src/side-navigation/side-navigation.scss

```diff
@@ -170,3 +171,41 @@
     }
   }
 }
+
+@mixin iui-side-navigation-submenu {
+  display: flex;
+  flex: 0 0 auto;
+  flex-direction: column;
+  height: 100%;
+  min-width: $iui-3xl * 2;
+  width: $iui-3xl * 4;
+  max-width: 50vw;
+  box-sizing: border-box;
+  padding: $iui-baseline $iui-sm;
+  overflow-x: hidden;
+  overflow-y: overlay;
+  resize: horizontal;
+  @media (prefers-reduced-motion: no-preference) {
```

Comment:
This animation looks odd.

Reply:
When it even works? We plan to change `max-width`?

Reply:
When you resize the whole window it has a delay.

Reply:
oh wow :O that is interesting

Reply:
I've removed it although my demo does not currently test / demonstrate the sub-menu being activated where this transition on max-width _might_ be necessary.

Reply:
I had included the transition on width to handle the submenu being activated when not present but it seems to be negatively effecting the resizing.  I've removed that and now included a region of CSS that copies the animation in/out classes we used on expandable block but I have not tested this yet.  Mayank had a cool way of testing this by linking the CSS to Storybook but I don't remember how to do that.